profiles/multisite_drupal_communities/modules/custom/multisite_og_button/multisite_og_button.test

```diff
@@ -16,126 +24,133 @@ class OGButtonTestCase extends DrupalWebTestCase {
 
     $this->drupalCreateContentType(array(
-    	'type' => 'group_content_b',
-      'name' => 'group_content_b', 
+        'type' => 'group_content_b',
+      'name' => 'group_content_b',
     ));
     og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_b');
-    
+
     $this->drupalCreateContentType(array(
-    	'type' => 'group_content_c',
-      'name' => 'group_content_c', 
+        'type' => 'group_content_c',
+      'name' => 'group_content_c',
     ));
     og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_c');
-    
+
     // Create a user who can edit groups.
     $this->users['admin'] = $this->drupalCreateUser(array(
       'access content',
       'administer blocks',
       'administer nodes',
       'administer users',
-			'administer content types',
+    'administer content types',
       'create groups content',
       'edit any groups content',
       'administer group',
     ));
-    $this->drupalLogin($this->users['admin']);    
-    
-    // Create some groups 
+    $this->drupalLogin($this->users['admin']);
+
+    // Create some groups.
     $settings = array();
     $settings['type'] = 'groups';
     $settings['title'] = 'Group A';
     $settings[OG_GROUP_FIELD][LANGUAGE_NONE][0]['value'] = 1;
     $this->groups['groupa'] = $this->drupalCreateNode($settings);
-    
+
     $settings = array();
     $settings['type'] = 'groups';
     $settings['title'] = 'Group B';
     $settings[OG_GROUP_FIELD][LANGUAGE_NONE][0]['value'] = 1;
     $this->groups['groupb'] = $this->drupalCreateNode($settings);
 
-    // Create a user who can edit group_content_a and group_content_b
+    // Create a user who can edit group_content_a and group_content_b.
     $this->users['usera'] = $this->drupalCreateUser(array(
       'access content',
       'create group_content_a content',
-			'create group_content_b content',                  
-      'edit any group_content_a content',                 
-      'edit any group_content_b content'
+    'create group_content_b content',
+      'edit any group_content_a content',
+      'edit any group_content_b content',
     ));
-    
+
     // Associate usera to the groupa.
-    $group = og_get_group('node',  $this->groups['groupa']->nid);
+    $group = og_get_group('node', $this->groups['groupa']->nid);
     og_group($group->gid, array(
-    	'entity' => $this->users['usera']
+        'entity' => $this->users['usera'],
     ));
-        
+
     $this->drupalLogout();
   }
-  
+
   /**
-   * This test will verify the following:
+   * Tests for OG button module.
+   *
+   * This test will verify the following :
    * - make sure the block is available
    * - the block is visible on a group page (context is available)
    * - the block displays the correct creation links (follows user permissions)
-   * - the block is invisible on a non-group page (context not available)
+   * - the block is invisible on a non-group page (context not available).
```

Comment:
The period at the end of the sentence is also missing from the other sentences in this list.